.gitignore

```diff
@@ -15,3 +15,6 @@ service-account-key.json
 # ignore media files in scripts folder
 scripts/*.mp3
 scripts/*.mp4
+
```

Comment:
Jupyterlite section should have:
- Overview: explaining the basics of the environment and listing the content of the section
- Actions:
-- Open
-- Run
-- Upload File
-- Copy Notebook

Reply:
Did something along those lines